frontend/src/pages/blocks/[hash].tsx

```diff
@@ -87,9 +87,25 @@ const BlockDetail: NextPage<Props> = (props) => {
           <Content
             size="medium"
             icon={<TransactionIcon style={{ width: "22px" }} />}
-            title={<h2>{t("common.receipts.receipts")}</h2>}
+            title={<h2>Receipts included in this block</h2>}
```

Comment:
What's our i18n goal? Should we just drop translations?

Reply:
No. For this moment it's just a demo. I'll add translations a bit later to this PR